client/my-sites/domains/domain-management/list/index.jsx

```diff
@@ -437,23 +355,25 @@ export class List extends React.Component {
 				this.setState( {
 					settingPrimaryDomain: false,
 					changePrimaryDomainModeEnabled: false,
-					notice: {
-						type: PRIMARY_DOMAIN_CHANGE_SUCCESS,
-						domainName: domain.name,
-						previousDomainName: currentPrimaryName,
-					},
 				} );
+
+				this.props.successNotice(
+					translate(
+						'Primary domain changed: all domains will redirect to {{em}}%(domainName)s{{/em}}.',
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **21** times:
`translate( 'Primary Domain changed: All domains will redirect to {{em}}%(domainName)s{{/em}}.' )` *ES Score: 9*